eng/pipelines/templates/variables/platform-list.csv

```diff
@@ -0,0 +1,6 @@
+win, 8|1.8
```

Comment:
This should not be in the pipelines/templates/variables folder. that is for our pipeline templates.

Why is this in a file at all? Is it supposed to stay in sync with our test matrix?